maliput-sys/src/api/rules/rules.h

```diff
@@ -335,6 +335,42 @@ rust::String Phase_id(const Phase& phase) {
   return phase.id().string();
 }
 
+std::unique_ptr<std::vector<DiscreteValueRuleState>> Phase_discrete_value_rule_states(const Phase& phase) {
+  const auto& discrete_value_rule_states_cpp = phase.discrete_value_rule_states();
+  std::vector<DiscreteValueRuleState> discrete_value_rule_states;
+  discrete_value_rule_states.reserve(discrete_value_rule_states_cpp.size());
+  for (const auto& discrete_value_rule_state_cpp : discrete_value_rule_states_cpp) {
+    DiscreteValueRuleState discrete_value_rule_state{discrete_value_rule_state_cpp.first.string(), std::make_unique<DiscreteValueRuleDiscreteValue>(discrete_value_rule_state_cpp.second)};
+    discrete_value_rule_states.push_back(std::move(discrete_value_rule_state));
+  }
+  return std::make_unique<std::vector<DiscreteValueRuleState>>(std::move(discrete_value_rule_states));
+}
+
+std::unique_ptr<std::vector<UniqueBulbId>> Phase_bulbs(const Phase& phase) {
```

Comment:
Can't you directly return a `std::unique_ptr<std::vector<std::unique_ptr<UniqueBulbId>>>` and avoid having a `ptr_from_unique_bulb_id` helper method?